ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -1,5 +1,152 @@
-module Ouroboros.Consensus.Storage.LedgerDB.OnDisk {-# DEPRECATED "Use Ouroboros.Consensus.Storage.LedgerDB instead" #-}
```

Comment:
So these warnings alone were not enough?

Reply:
So if things only had moved around, they would probably have been. However as a couple of types changes their name, they were undiscoverable.